components/BattleScreen/Menu/AttackMenu.js

```diff
@@ -12,83 +12,139 @@ const AttackMenu = ({ onAttackSelection, onBackButtonClick, attacks }) => {
 	const [hoveredButton, setHoveredButton] = React.useState(null);
 
 	return (
-		<AttackMenuContainer>
-			{attacks.map((attack, index) => (
-				<AttackButton
-					key={index}
-					onClick={() => {
-						handleAttackClick(index);
-						playSound("menuSound");
-					}}
+		<>
+			<BackgroundImageContainer />
+			<AttackMenuContainer>
+				{attacks.map((attack, index) => (
+					<AttackButton
+						key={index}
+						onClick={() => {
+							handleAttackClick(index);
+							playSound("menuSound");
+						}}
+						onMouseEnter={() => {
+							playSound("menuSound");
+							setHoveredButton(index + 1);
+						}}
+						onMouseLeave={() => {
+							stopSound("menuSound");
+							setHoveredButton(null);
+						}}
+						type={attack.type}
```

Comment:
please do not use the type prop for any custom values, as it is a prop that has a meaning for [HTML buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button?retiredLocale=de#attributes).
E.g. this button should be of type button.

```suggestion
						type="button"
						attackType={attack.type}
```

Reply:
I have changed, thank you!